plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts

```diff
@@ -88,32 +88,42 @@ export function createSentryCreateProjectAction(options: { config: Config }) {
         throw new InputError(`No valid sentry token given`);
       }
 
-      const response = await fetch(
-        `https://sentry.io/api/0/teams/${organizationSlug}/${teamSlug}/projects/`,
-        {
-          method: 'POST',
-          headers: {
-            Authorization: `Bearer ${token}`,
-            'Content-Type': 'application/json',
-          },
-          body: JSON.stringify(body),
-        },
-      );
+      const { result } = await ctx.checkpoint({
+        key: `create.project.${organizationSlug}.${teamSlug}`,
+        // eslint-disable-next-line no-loop-func
```

Comment:
Do we need this? Don't think there is a loop right?

Reply:
Oops 👀